.prettierrc

```diff
@@ -6,5 +6,7 @@
   "printWidth": 120,
   "jsxBracketSameLine": true,
   "jsxSingleQuote": true,
-  "endOfLine": "auto"
+  "endOfLine": "auto",
+  "importOrder": ["^([^(.|api\\-client|build\\-config)])", "api-client", "build-configs", "translations", "^./*"],
```

Comment:
I'm not sure whether we want to add these rules for manual sorting. I suspect that we are missing maybe some edgecases or will miss to adjust this regex as soon as we add a new internal dependency.

For simplicity I would just tread our dependencies as every other dependencies.

Reply:
i think we should differ between 3rd party library like node modules and our symlinks from the monorepo and local imports.

For me it would be okay to add a new regex rules part if we have a new internal lib. I don't think this happens too often.
I like the solution here. i would just avoid to make new lines except for local imports if that it possible

Reply:
As there are different opinions about this (see discussions/votes on Mattermost), I decided to do it this way:

everything external dependencies
(newline)
our dependencies
(newline)
local imports

Reply:
If anyone is unhappy with this solution we should talk about this on Wednesday and discuss this together, but I think it does not really matter.